.github/workflows/ci.yml

```
@@ -57,4 +57,12 @@ jobs:
       - name: 빌드 테스트
-        run: ./gradlew clean build -x test
\ No newline at end of file
+        run: ./gradlew clean build --stacktrace
+
+      - name: PR에 코드 커버리지 측정 레포트 생성
+        id: jacoco
+        uses: madrapps/jacoco-report@v1.7.2
+        with:
+          paths: |
+            ${{ github.workspace }}/**/build/jacoco/index.xml
+          token: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)